frontend/src/GetLink.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {useEffect, useState} from 'react';
+import {LinkModel} from "./LinkModel";
+import axios from "axios";
+
+function GetLink() {
```

Comment:
React-Komponenten sollten eher nach dem benannt sein, dass sie visualisieren. `GetLink` ist hingegen eher eine Aktion / Handlung / Verb.
Da nun `Link` schon Dein Bezeichner für das Model ist (gut so, bitte beibehalten) müssen wir für den Namen der Komponente kreativer werden.
Vorschlag: `LinkCard` oder irgendetwas in der Richtung

Reply:
Vielen Dank für den Vorschlag, Daniel!